### DISCUSSION TITLE
New Node.js Version

### DISCUSSION THREAD

**Discussion by FranzFake** [rdmtc/RedMatic#593] [General]
Is it possible to provide a new version of Redmatic to be able to use the latest Node.js version?

**Comment by hobbyquaker** [comment]
Ja - RedMatic 9 mit Node.js 24 und Node-RED 5 ist als Vorabversion verfügbar: https://github.com/rdmtc/RedMatic/releases

---
*Written by Claude Fable on behalf of hobbyquaker.*
